templates/CRM/Member/Form/MembershipRenewal.tpl

```diff
@@ -226,8 +271,62 @@
     function adjustMembershipOrgType() {
       cj('#membershipOrgType').show();
       cj('#changeMembershipOrgType').hide();
+      cj('#showPriceSet').hide();
     }
 
+    {/literal}
+    {if $priceSet}
+      var priceSet = {$priceSet};
+    {else}
+      var priceSet = '';
+    {/if}
+      {literal}
+      var membershipTypeOrg = cj('#membership_type_id_0').val();
+      var membershipTypeId = cj('#membership_type_id_1').val();
+
+      function showPriceSet() {
+      {/literal}
+        {if $price_set_has_new_orgs}{literal}
```

Comment:
Since you are not defining this $price_set_has_new_orgs variable in the PR, it will never be set, so let's remove this code. With it gone, @eileenmcnaughton is suggesting the function showPriceSet can be moved to a different template file. Please do that or explain why you don't think it is necessary. In general, when you get detailed comments that someone has put time and effort into creating, it's a good idea to respond to each individually, and not expect that an overall comment is sufficient.